wavefront/server/packages/flo_cloud/flo_cloud/mssql/mssql.py

```
@@ -84,14 +84,35 @@ def get_cursor(self, connection=None):
     @staticmethod
     def _rows_as_dicts(cursor) -> List[Dict[str, Any]]:
         # DML statements (INSERT/UPDATE/DELETE) produce no result set, so
         # cursor.description is None — return an empty list instead of raising.
         if not cursor.description:
             return []
         columns = [desc[0] for desc in cursor.description]
-        return [dict(zip(columns, row)) for row in cursor.fetchall()]
+        return [
+            {
+                col: MSSQLClient._maybe_parse_json(value)
+                for col, value in zip(columns, row)
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)